src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -343,19 +344,23 @@ private function validateNamespace($name)
     {
         $alias = null;
         $aliasLength = strpos($name, ':');
-        if (false === $aliasLength) {
+
+        if ($aliasLength === false) {
             return;
-        } 
-        
-        $alias = substr($name, 0, $aliasLength);
+        } else {
```

Comment:
no need for an else. when the `if` is entered, we return. not having the else makes the code more readable as you see that we either already stopped, or we get to this line.